werkzeug/wrappers.py

```diff
@@ -1443,12 +1484,43 @@ def make_conditional(self, request_or_environ):
             # wsgiref.
             if 'date' not in self.headers:
                 self.headers['Date'] = http_date()
+            if accept_ranges is True:
+                accept_ranges = "bytes"
+            elif accept_ranges is None:
+                accept_ranges = "none"
+            elif not isinstance(accept_ranges, string_types):
+                accept_ranges = None
+            if accept_ranges is not None:
+                self.headers['Accept-Ranges'] = accept_ranges
+                if self._is_range_request_processable(environ) and complete_length is not None:
```

Comment:
I think this massively indented block should be moved into its own helper function. Then it will be possible to get rid of the nested ifs by using `return`.